packages/client/test/client.spec.js

```diff
@@ -244,21 +262,64 @@ test('execute with delegations', async () => {
 
   const [e1] = await connection.execute(await add.delegate())
 
-  assert.deepEqual(e1, {
-    error: true,
-    name: 'UnknownDIDError',
-    message: `DID ${bob.did()} has no account`,
-    did: bob.did(),
+  assert.deepEqual(e1.out, {
+    error: {
+      error: true,
+      name: 'UnknownDIDError',
+      message: `DID ${bob.did()} has no account`,
+      did: bob.did(),
+    },
   })
 
   // fake register alice
   service.access.accounts.register(bob.did(), 'did:email:bob@web.mail', car.cid)
 
   const [r1] = await connection.execute(await add.delegate())
-  assert.deepEqual(r1, {
-    with: bob.did(),
-    link: car.cid,
-    status: 'upload',
-    url: 'http://localhost:9090/',
+  assert.deepEqual(r1.out, {
+    ok: {
+      with: bob.did(),
+      link: car.cid,
+      status: 'upload',
+      url: 'http://localhost:9090/',
+    },
+  })
+})
+
+test('decode error', async () => {
+  const client = Client.connect({
+    id: w3,
+    channel,
+    codec: Codec.outbound({
+      encoders: {
+        'application/car': CAR.request,
+      },
+      decoders: {
+        'application/car+receipt': CAR.response,
```

Comment:
hmm `application/car` had been used all over the place, but looks like IPFS gateway now sets `application/vnd.ipld.car;version=1` and so should we.

Complication however here is that our codecs also used to set this as content type before this change so not recognizing it will break existing clients. I created a followup issue for this https://github.com/web3-storage/ucanto/issues/269